CHANGES.rst

```diff
@@ -2,6 +2,1614 @@
 Cython Changelog
 ================
 
+3.0.0 unified release notes
+===========================
+
+Cython 3.0.0 has been a very large effort that cleaned up many old warts,
+introduced many new features, and introduces a couple of intentional
+behaviour changes, even though the goal remained to stay compatible as
+much as possible with Cython 0.29.x. For details, see the `migration guide`_.
+
+.. _`migration guide`: https://cython.readthedocs.io/en/latest/src/userguide/migrating_to_cy30.html
+
+As the development was spread out over several years, a lot of things have
+happened in the meantime. Many crucial bugfixes and some features were
+backported to 0.29.x and are not strictly speaking "new" in Cyton 3.0.0.
+We mark affected lines below with †.
```

Comment:
Note that none of the backports have been mentioned yet, nor marked accordingly. This was just an idea when I started this, perhaps it's too much effort and we can simply drop it.